blockly-svelte/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "svelte-app",
+  "version": "1.0.0",
+  "devDependencies": {
+    "npm-run-all": "^4.1.5",
```

Comment:
What's npm-run-all for?

Reply:
It is for the autobuild development script `"dev": "run-p start:dev autobuild"`, which is suggested by svelte's scaffolding template.